source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -17,6 +17,8 @@ @implementation ACRChoiceSetViewDataSource
     NSIndexPath *_lastSelectedIndexPath;
     NSMutableSet *_defaultValuesSet;
     NSArray *_defaultValuesArray;
+    CGFloat _padding;
+    CGFloat _accessoryViewWidth;
```

Comment:
These should just be constants, instead of fields on every instance of the class.

Same for the analogous fields on other classes.